src/main/java/org/opentripplanner/transit/model/site/GroupStopBuilder.java

```diff
@@ -69,24 +68,26 @@ public I18NString name() {
   }
 
   public GroupStopBuilder addLocation(StopLocation location) {
+    if (
+      !(
+        location.getStopType() == StopType.REGULAR ||
+        location.getStopType() == StopType.FLEXIBLE_AREA
+      )
+    ) {
+      throw new RuntimeException(
+        "Unsupported location for GroupStop. Must be Regular or FlexibleArea."
```

Comment:
These messages have a habit of becoming outdated. Can you use placeholders and something like `GroupStop.class.getSimpleName` and the enum values?

Reply:
Really? I don't see it.

Reply:
Locally. Don't hold your breath.